ledger/catchupaccessor.go

```diff
@@ -820,6 +867,12 @@ func (c *catchpointCatchupAccessorImpl) VerifyCatchpoint(ctx context.Context, bl
 		if err != nil {
 			return fmt.Errorf("unable to get accounts totals: %v", err)
 		}
+
+		rawStateProofVerificationContext, err = CatchpointStateProofVerification(ctx, tx)
```

Comment:
if the file version is less than 7 the table will be empty, do we have a test for this?

Reply:
test: build new binary that support catchpoint v7, and catchup betanet (testnet) that is catchpooint v6